### PR TITLE
e2e/framework: remove direct imports to /pkg/kubelet/...

### PR DESCRIPTION
**Pull request by gavinfish**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

e2e/framework: remove direct import to pkg/kubelet/dockershim/metrics and pkg/kubelet/metrics.

**Which issue(s) this PR fixes**:

Ref #74352

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
none
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
none
```

### CODE REVIEW

File: test/e2e/framework/metrics/kubelet_metrics.go [hunk 1]
```diff
@@ -29,13 +29,19 @@ import (
...
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/component-base/metrics/testutil"
-	dockermetrics "k8s.io/kubernetes/pkg/kubelet/dockershim/metrics"
-	kubeletmetrics "k8s.io/kubernetes/pkg/kubelet/metrics"
 	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
 )
 
 const (
 	proxyTimeout = 2 * time.Minute
+	// DockerOperationsLatencyKey is the key for the operation latency metrics.
```
Comment (neolit123) [inline test/e2e/framework/metrics/kubelet_metrics.go]: please add comments where these constants are coming from and group them.

```
// Taken from k8s.io/kubernetes/pkg/kubelet/metrics
...
// k8s.io/kubernetes/pkg/kubelet/dockershim/metrics
...
```

Comment (neolit123) [reply test/e2e/framework/metrics/kubelet_metrics.go]: also the constants can now be local (lowercase)

**Comment by tanjunchen** [comment]
Thanks
/lgtm

File: test/e2e/framework/metrics/kubelet_metrics.go [hunk 2]
```diff
@@ -29,13 +29,26 @@ import (
...
+	// Taken from k8s.io/kubernetes/pkg/kubelet/metrics
+	podWorkerDurationKey = "pod_worker_duration_seconds"
+	// Taken from k8s.io/kubernetes/pkg/kubelet/metrics
+	podStartDurationKey = "pod_start_duration_seconds"
+	// Taken from k8s.io/kubernetes/pkg/kubelet/metrics
+	cgroupManagerOperationsKey = "cgroup_manager_duration_seconds"
+	// Taken from k8s.io/kubernetes/pkg/kubelet/metrics
+	podWorkerStartDurationKey = "pod_worker_start_duration_seconds"
+	// Taken from k8s.io/kubernetes/pkg/kubelet/metrics
+	pLEGRelistDurationKey = "pleg_relist_duration_seconds"
```
Comment (oomichi) [inline test/e2e/framework/metrics/kubelet_metrics.go]: nit: I guess the first "p" is for local, but I feel `plegRelistDurationKey` seems more readable.

Comment (oomichi) [reply test/e2e/framework/metrics/kubelet_metrics.go]: /cc @oomichi

Comment (gavinfish) [reply test/e2e/framework/metrics/kubelet_metrics.go]: Done.

**Review by neolit123 — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**